src/pages/impact/index.tsx

```diff
@@ -0,0 +1,155 @@
+        },
+        {
+          number: "13.4k",
+          secondaryNumber: 6,
+          statement: "Annual users",
+        },
+        {
+          number: "111",
+          statement: "New resources",
+        },
+      ]}
+      theme="white"
+    />
+    <StatsBarBlock
+      title="ShelterConnect"
+      subtitle="Delivering free, turn-key WiFi solutions to shelters and transitional housing facilities with key partners. Learn more about this program."
+      barStatCards={[
+        {
+          bars: [
+            {
+              label: 2019,
+              statistic: 14,
+            },
+            {
+              label: 2020,
+              statistic: 17,
+            },
+            {
+              label: 2021,
+              statistic: 21,
+            },
+          ],
+          title: "Shelters Wired",
+          subtitle: "+4 Y0Y",
+          orientation: "vertical",
+        },
+        {
+          bars: [
+            {
+              label: "PEAK",
+              statistic: 14,
+            },
+            {
+              label: "TARGET",
+              statistic: 17,
+            },
+            {
+              label: 2020,
+              statistic: 21,
+            },
+          ],
+          title: "People serviced daily",
+          subtitle: "Daytime",
+          orientation: "horizontal",
+        },
+        {
+          bars: [
+            {
+              label: "PEAK",
+              statistic: 14,
+            },
+            {
+              label: "TARGET",
+              statistic: 17,
+            },
+            {
+              label: 2020,
+              statistic: 21,
+            },
+          ],
+          title: "People serviced daily",
+          subtitle: "Nighttime",
+          orientation: "vertical",
+        },
+      ]}
+      theme="gray"
+      titleColor="blue"
+    />
+    <StatsBlock
+      title="Volunteers"
+      subtitle="As an all-volunteer organization ShelterTech relies on volunteers to support digital equity. Learn more about volunteering."
+      statCards={[
+        {
+          number: "107",
+          secondaryNumber: 6,
+          statement: "Active volunteers",
+        },
+        {
+          number: "13",
+          secondaryNumber: 6,
+          statement: "Community Representatives",
+        },
+        {
+          number: "20",
+          secondaryNumber: 6,
+          statement: "Corporate volunteer organizations",
+        },
+      ]}
+      theme="white"
+    />
+    <StatsBlock
+      title="Funding"
+      subtitle="As an all-volunteer organization ShelterTech relies on volunteers to support digital equity. Help us reach our next funding goal."
+      statCards={[
+        {
+          number: "$79k",
+          secondaryNumber: 6,
+          statement: "Annual funding",
+        },
+        {
+          number: "$2m",
+          secondaryNumber: 6,
+          statement: "in in-kind contributions",
+        },
+      ]}
```

Comment:
It looks like the horizontal alignment of these cards is a bit off when there are only two cards. In the Figma design, it looks like the cards should be in the exact same places as when there are three cards, so we probably need to get rid of some of styling that causes the cards to expand. Could you update that as well?